addon/templates/components/uni-tooltip.hbs

```diff
@@ -0,0 +1,15 @@
+<span class="uni-tooltip__icon">
+    {{#if informationIcon}}
```

Comment:
I believe it is better to name this ```isInformative``` than ```informationIcon```.

Booleans should always start by ```has```or ```is```